### DISCUSSION TITLE
Can i use koreader for windows phone 10?

### DISCUSSION THREAD

**Discussion by ichinoseayaka** [koreader/koreader#11607] [Q&A] [answered]
I have a lumia phone and i want to compile and run koreader in my phone. Can i install it?

**Comment by Frenzie** [comment]
As in an old Windows Phone from a decade ago? I'm not sure if that even supports "regular" apps.

**Reply by ichinoseayaka** [reply]
Yes. It's a lumia 730 phone

**Reply by Frenzie** [reply, marked as answer]
That won't be possible, no.

**Reply by ichinoseayaka** [reply]
okay. I know. I will sell this phone and buy a old android phone
